public/diagnostic.js

```diff
@@ -23,3 +23,31 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+
+const Order = Ember.Object.extend({
+  orderPrice : Ember.computed('price','quantity', function(){
+    return this.get('price') * this.get('quantity');
+  })
+});
+
+const Cart = Ember.Object.extend({
+  orders: [],
+  addToCart :() =>{
+  },
+  totalPrice : Ember.computed('orders.@each.price', function(){
+    let orders = this.get('orders');
+    return
+
+  })
+});
+
+
+let item = Cart.create({
+  orders: [
+    Ember.Object.create({price: 5,quantity: 2}),
+    Ember.Object.create({price: 20,quantity: 1}),
+    Ember.Object.create({price: 8,quantity: 3}),
+
+  ]
+});
```

Comment:
You don't need to use `Ember.Object.create` here, after you finish writing `addToCart`, then this will look like `item.addToCart(Order.create({price: 5, quantity: 2}));`